functions/Dismount-DbaDatabase.ps1

```diff
@@ -194,13 +194,15 @@ function Dismount-DbaDatabase {
 
             If ($Pscmdlet.ShouldProcess($server, "Detaching $db on $server")) {
                 try {
+                    $dbID = $db.ID
                     $server.DetachDatabase($db.Name, $UpdateStatistics)
 
                     [pscustomobject]@{
                         ComputerName = $server.ComputerName
                         InstanceName = $server.ServiceName
                         SqlInstance  = $server.DomainInstanceName
                         Database     = $db.name
+                        DatabaseID   = $dbID
```

Comment:
I think this should be `Id` instead of `DatabaseID`, as ID is a natural part of SMO but we use PS notation, so Id 🤔